content/sips/sip-420.md

```diff
@@ -0,0 +1,105 @@
+---
+sip: 420
+title: Protocol Owned SNAX
+network: All
```

Comment:
can you please use one of these
```
(['Ethereum', 'Optimism', 'Ethereum & Optimism','Base','Ethereum, Optimism & Base','Ethereum & Base','Optimism & Base','Arbitrum','Ethereum, Optimism, Base & Arbitrum','Ethereum & Arbitrum','Arbitrum & Base'])
```

Reply:
This structure seems insane tbh, even more so because there is no option for Snaxchain.